common.gypi

```diff
@@ -503,7 +513,18 @@
         'ldflags': [
           '-Wl,--export-dynamic',
         ],
-      }]
+      }],
+      ['node_shared_openssl!="true"', {
```

Comment:
@refack You suggested that to avoid OpenSSL flags being passed to icu (and thus triggering an icu rebuild when switching between shared/non-shared openssl, a change that doesn't effect icu), I could "Add `OPENSSL_THREADS` to the else clause.", but I'm not sure what you mean by else clause here. Can you elaborate, and I'll make the change and test it?

Reply:
cf. https://github.com/nodejs/node/issues/25135